src/router/index.js

```diff
@@ -21,4 +22,15 @@ const Router = new VueRouter({
     routes
 });
 
+Router.afterEach(to => {
+    Vue.gtm.trackView(
+        to.path,
+        to.name,
+        {
+            cid: ga.getSessionId() || ga.createSessionId(),
```

Comment:
I don't think these files were checked in?

Reply:
@wildtron is `./analytics` an already existing file?

Reply:
With your latest code, we just need createSessionId()?

Reply:
Yeah. I'll have to resolve every comments first in FPD so we can finalize everything that's going to boilerplate.

Reply:
Resolved: https://github.com/anyTV/quasar-boilerplate/pull/32/commits/af5f421f6e2a99a589f082938cd1dacaf2d66365